lib/Agrammon/UI/CommandLine.pm6

```diff
@@ -28,13 +28,14 @@ my %*SUB-MAIN-OPTS =
   :named-anywhere,    # allow named variables at any location
 ;
 
+subset OptionalExistingFile of Str where { .IO.e or note("No such file $_") && exit 1 }
```

Comment:
This has been added, but isn't being used

Reply:
Ah, yes, decided I can't use it ... see below. Removed.